frontend/src/components/common/Filter/components/FilterMultiPanel/components/FilterPanelList/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { IconNames } from "@blueprintjs/icons";
+import { Box, List, ListItem, ListItemText } from "@material-ui/core";
```

Comment:
SDS has List and ListItem too! Can we see if we can just use those instead? Thank you!